blaze/expr/arithmetic.py

```diff
@@ -50,6 +50,55 @@ def name(o):
         return None
 
 
+class Op(ElemWise):
+    def __init__(self, ops):
+        if not isinstance(ops, (list, tuple,)):
```

Comment:
That actually seems nicer than a unary/binary split. This could let us get some fun things like ifexprs in the mix maybe.